src/components/Dropdown.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <!-- Copied and modified from Vuetensils to match the existing bulma implementation -->
+  <!-- https://github.com/AustinGil/vuetensils/blob/production/src/components/VDropdown/VDropdown.vue -->
+  <div
+    class="navbar-item has-dropdown is-hoverable"
+    @mouseenter="isHovered = true"
+    @mouseleave="isHovered = false"
+    @focus="isFocused = true"
+    @blur="isFocused = false"
+    @focusout="onFocusout"
+  >
+    <div
+      :aria-expanded="!!isHovered || !!isFocused"
+      aria-haspopup="true"
+      class="navbar-link is-arrowless"
+      role="button"
+      tabindex="0"
+      @keydown.enter="isFocused = !isFocused"
+      @keydown.space.prevent="isFocused = !isFocused"
+      @click="isFocused = !isFocused"
+      @focus="isFocused = true"
+    >
+      <slot name="trigger">
+        {{ text }}
+        <i class="icon caret-down" />
+      </slot>
+    </div>
+
+    <div
+      class="navbar-dropdown"
+      :class="{ visible: !!isHovered || !!isFocused }"
+      role="menu"
+    >
+      <slot :onFocus="onFocus" />
+    </div>
+  </div>
+</template>
+
+<script>
+/**
+ * Mimics the behavior of the WordPress.org menubar. Creates an accessible focusable trigger element that
+ * when focused, opens an accessible dropdown menu.
+ *
+ * This does not fully follow the ARIA specification/recommendation for accessible dropdown in order to stay
+ * consistent with the WordPress.org menubar. For example, it does not implement arrow key navigation,
+ * instead just allowing for tab navigation through each item.
+ */
+const Dropdown = {
+  name: 'Dropdown',
+  props: {
+    /**
+     * The trigger element text.
+     */
+    text: {
+      type: String,
+      default: '',
+    },
+  },
+  data: () => ({
+    isHovered: false,
+    isFocused: false,
+  }),
+  mounted() {
+    document.addEventListener('click', this.onClickout)
+  },
+  beforeDestroy() {
+    document.removeEventListener('click', this.onClickout)
+  },
+  beforeUnmount() {
+    document.removeEventListener('click', this.onClickout)
+  },
+  methods: {
+    onClickout(e) {
+      if (!this.$el.contains(e.target)) {
+        this.isFocused = false
+      }
+    },
+    onFocusout(event) {
+      if (!this.$el.contains(event.relatedTarget)) {
+        this.isFocused = false
+      }
+    },
+    onFocus() {
+      this.isFocused = true
+    },
+  },
+}
+
+export default Dropdown
+</script>
+
+<style lang="scss">
+.navbar-dropdown {
```

Comment:
Can we change this in the original scss navbar file insidethe styles folder?

Reply:
Done in d0c6ebe